### PR TITLE
[DONT MERGE] Adding identity and quota usages and quotas changes

### PR DESCRIPTION
**Pull request by nkalpana**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/machinelearningservices/resource-manager/readme.md [hunk 1]
```diff
@@ -24,6 +24,22 @@ To see additional help and options, run:
 
 These are the global settings for the Machine Learning Services API.
 
+``` yaml
+openapi-type: arm
+tag: package-2020-04-01
+```
```
Comment (ArcturusZhang) [inline specification/machinelearningservices/resource-manager/readme.md:30]: Well we should not adding a new snippet here, instead we need to modify the existing tag to `package-2020-04-01`.
Therefore could we please remove this, and change the `package-2020-03-01` to `package-2020-04-01` below?

File: specification/machinelearningservices/resource-manager/readme.md [hunk 2]
```diff
@@ -24,6 +24,22 @@ To see additional help and options, run:
...
+
+```yaml $(tag) == 'package-2020-04-01'
+input-file:
+  - Microsoft.MachineLearningServices/stable/2020-04-01/machineLearningServices.json
+```
+
+
 ``` yaml
 openapi-type: arm
 tag: package-2020-03-01
```
Comment (ArcturusZhang) [inline specification/machinelearningservices/resource-manager/readme.md:45]: Please change this to 
```suggestion
tag: package-2020-04-01
```

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]
A few comments

File: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-04-01/examples/ListSkus.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
+{
+  "parameters": {
+    "api-version": "2020-03-01",
```
Comment (ArcturusZhang) [inline specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-04-01/examples/ListSkus.json:3]: Seems this should be `2020-04-01`? Please double check all other examples for similar issues
```suggestion
    "api-version": "2020-04-01",
```

**Review by ArcturusZhang — COMMENTED** [review]
Hi @nkalpana a swagger for `2020-04-01/machinelearningservices.json` has been merged into master, could you please do a merge back from master and resolve the possible conflicts?

**Comment by ravbhatnagar** [comment]
@nkalpana - please let us know if a review on this PR is still needed. removing ARM review labels for now.

**Comment by ArcturusZhang** [comment]
Close this PR since there are too many unresolved conflicts and the author not responding for quite a long time.
Please feel free to open a new PR based on the latest master branch with the new changes when you come back @nkalpana

**Pull request closed without merging**